src/components/app/BottomBox.css

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.bottom-box {
+  display: flex;
+  height: 100%;
+  flex-flow: column nowrap;
+}
+
+.bottom-main {
+  position: relative;
+  display: flex;
+  min-height: 0;
+  flex: 1;
+  flex-flow: column nowrap;
+  background: var(--grey-20);
+}
+
+.bottom-box-bar {
+  display: flex;
+  overflow: hidden;
+  height: 24px;
+  flex-flow: row nowrap;
+  align-items: center;
+  justify-content: space-between;
+  background: var(--grey-10);
+  box-shadow: inset 0 -1px var(--grey-30);
+  line-height: 18px;
+}
```

Comment:
Thanks for the link. I'm not very familiar with CSS coding standards.

Reply:
Hmm, I can't think of a good grouping. I added an empty line between justify-content and background, but then couldn't think of a good label for the "background, box-shadow, line-height" part. And I'm not sure that as a reader I would benefit from such labels. I feel like if this is something we want to have in general, it might be better to have our formatter add these labels automatically. Then they'd be consistent.